extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/HibernateMetadataWithInjectionTest.java

```diff
@@ -1,17 +1,17 @@
 package io.quarkus.hibernate.orm;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.*;
```

Comment:
We don't use star imports in Quarkus, please change here and the other files you added it to